.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
This config auto-formats all code using pretty-quick whenever there is a commit obviating the need for eslint to do this.